### ISSUE TITLE
BAD RESULT

### ISSUE THREAD

**Issue by BruceWeiii** [fjiang9/NKF-AEC#12]
Hello, I have replicated your work but get  bad result.  Could you describe your training process? I modified the test code for training and found that the results were poor.

**Comment by XJWesley** [comment]
Yes, I got the same bad result. I want to know you have made its result good. we can consult it together.

**Comment by debasish-mihup** [comment]
@XJWesley You got bad result using your own trained model or you are getting same bad results with the provided checkpoints as well?

**Comment by XJWesley** [comment]
> @XJWesley You got bad result using your own trained model or you are getting same bad results with the provided checkpoints as well?

Just my own trained model!I can't get the same result

**Comment by kafan1986** [comment]
> > @XJWesley You got bad result using your own trained model or you are getting same bad results with the provided checkpoints as well?
> 
> Just my own trained model!I can't get the same result

Do you have the codebase that you used for training? Would have liked to train it my end as well.
